### DISCUSSION TITLE
Custom background color for input checkbox type

### DISCUSSION THREAD

**Discussion by yuxxeun** [tailwindlabs/tailwindcss#12351] [Help]
How to customizing `background-color` on input checkbox with Tailwindcss? 
I've use `text-(color)-(value)` , `bg-(color)-(value)` but still doesnt change

![image](https://github.com/tailwindlabs/tailwindcss/assets/50067865/53b1c0df-9450-475e-a699-66996824a03e)

**Comment by adydetra** [comment]
Use [Accent Color](https://tailwindcss.com/docs/accent-color)

[Preview](https://play.tailwindcss.com/oUGUGYS2Wm) on Tailwind Play

```html
<section class="flex flex-col">
  <div>
    <input type="checkbox" class="bg-red-500 mx-1" id="checkbox1"/>
    <label for="checkbox1">Checkbox Label 1</label>
  </div>

  <div>
    <input type="checkbox" class="accent-pink-500 mx-1" id="checkbox2"/>
    <label for="checkbox2" class="text-[#00E321]">Checkbox Label 2</label>
  </div>

  <div>
    <input type="checkbox" class="accent-[#000] mx-1" id="checkbox3"/>
    <label for="checkbox3" class="text-[#AAA312]">Checkbox Label 3</label>
  </div>
</section>
```

![image](https://github.com/tailwindlabs/tailwindcss/assets/108399634/6564b543-185c-4556-a675-d65edb2eaf6c)

**Reply by mohamedyasser27** [reply]
is there a way to change the tick color?

or do i handle it manually using `appearance-none` and rebuild the checkbox?
